src/api/fetchThemeProducts.ts

```diff
@@ -0,0 +1,32 @@
+import type { GoodsData } from '@/types';
+
+import instance from './api';
+
+interface FetchThemeProductsResponse {
+  products: GoodsData[];
+  nextPageToken: string | null;
+  pageInfo: {
+    totalResults: number;
+    resultsPerPage: number;
+  };
+}
+
+export const fetchThemeProducts = async (
+  themeKey: string,
+  pageToken?: string,
+  maxResults: number = 20,
+): Promise<FetchThemeProductsResponse> => {
+  const params: { pageToken?: string; maxResults: number } = { maxResults };
+  if (pageToken) {
+    params.pageToken = pageToken;
+  }
+
+  const res = await instance.get<FetchThemeProductsResponse>(
+    `/api/v1/themes/${themeKey}/products`,
+    {
+      params,
+    },
+  );
+
+  return res.data;
```

Comment:
```suggestion
  const params: { pageToken?: string; maxResults: number } = { maxResults };
  
  if (pageToken) {
    params.pageToken = pageToken;
  }

  const res = await instance.get<FetchThemeProductsResponse>(
    `/api/v1/themes/${themeKey}/products`,
    {
      params,
    },
  );

  return res.data;
```
코드 전체적으로 줄바꿈에 대한 통일된 코드 컨벤션이 있으면 좋을 것 같아요.
예를 들어 저는 if문 전, 함수호출 전, return문 전에는 줄바꿈을 한번씩 해주는 편입니다.